File: add/codesnippet/VisualBasic/p-system.net.networkinfo_114_1.vb
    Public Shared Sub ShowRedirectsData() 
        Dim properties As IPGlobalProperties = IPGlobalProperties.GetIPGlobalProperties()
        Dim statistics As IcmpV4Statistics = properties.GetIcmpV4Statistics()
        Console.WriteLine("  Redirects ........................... Sent: {0,-10}   Received: {1,-10}", statistics.RedirectsSent, statistics.RedirectsReceived)
    
    End Sub 'ShowRedirectsData
    

File: add/codesnippet/CSharp/p-system.net.networkinfo_114_1.cs
        public static void ShowRedirectsData()
        {
            IPGlobalProperties properties = IPGlobalProperties.GetIPGlobalProperties();
            IcmpV4Statistics statistics = properties.GetIcmpV4Statistics();
            Console.WriteLine("  Redirects ........................... Sent: {0,-10}   Received: {1,-10}",
                statistics.RedirectsSent, statistics.RedirectsReceived);  
        }